test/tests/change_file_content.rs

```diff
@@ -8,7 +6,7 @@ use lockbook_core::model::api::NewAccountRequest;
 use lockbook_core::model::api::{ChangeFileContentError, ChangeFileContentRequest};
 
 #[macro_use]
-mod utils;
+pub mod utils;
```

Comment:
note: tests are built as their own modules; shared functions that are unused in any of the tests produced unused warnings even if used in other tests. Making this public suppresses those warnings